lib/mobility-core/src/Kernel/Tools/Slack/Middleware.hs

```diff
@@ -58,14 +62,15 @@ notifyApiCallOnSlack :: (HasField "slackEnv" env SI.SlackEnv) => env -> ByteStri
 notifyApiCallOnSlack appEnv body path sc = do
   let title = "*API TRIGGERED: *" <> cs path
       body' = "```" <> cs body <> "```"
-      slackEnv = bool (appEnv.slackEnv {SI.channel = sc}) appEnv.slackEnv (sc == "default")
+      updatedSlackEnv = (getField @"slackEnv" appEnv) {SI.channel = sc}
+      slackEnv = bool (getField @"slackEnv" appEnv) updatedSlackEnv (sc == "default")
   Slack.notifyOnSlackIO slackEnv title body' Nothing Nothing
 
 notifyApiCallResponseOnSlack :: (HasField "slackEnv" env SI.SlackEnv) => Text -> env -> Wai.Response -> Maybe Chat.PostMsgRsp -> IO (Slack.Response Chat.PostMsgRsp)
 notifyApiCallResponseOnSlack sc appEnv resp st = do
   let (status, _, _) = Wai.responseToStream resp
       code = HTTP.statusCode status
-      slackEnv = bool (appEnv.slackEnv {SI.channel = sc}) appEnv.slackEnv (sc == "default")
+      slackEnv = bool ((getField @"slackEnv" appEnv) {SI.channel = sc}) (getField @"slackEnv" appEnv) (sc == "default")
```

Comment:
this is third possible syntax:

- (getField @"slackEnv" appEnv) 
- (field' @"slackEnv" appEnv) 
- (#slackEnv appEnv) 
can we use the same everywhere?